x/evmutil/keeper/conversion_evm_native_bep3.go

```diff
@@ -0,0 +1,55 @@
+package keeper
+
+import (
+	"math/big"
+
+	errorsmod "cosmossdk.io/errors"
+	"github.com/kava-labs/kava/x/evmutil/types"
+)
+
+var (
+	bep3Denoms = map[string]bool{
+		"bnb":  true,
+		"busd": true,
+		"btcb": true,
+		"xrpb": true,
+	}
+
+	bep3ConversionFactor = new(big.Int).Exp(big.NewInt(10), big.NewInt(10), nil)
+)
+
+func isBep3Asset(denom string) bool {
+	return bep3Denoms[denom]
+}
+
+// convertBep3CoinAmountToERC20Amount converts a bep3 coin amount with 8 decimals
+// to the equivalent ERC20 token with 18 decimals.
+func convertBep3CoinAmountToERC20Amount(amount *big.Int) *big.Int {
```

Comment:
Just a personal preference and no need to change, but I like named return arguments for functions like this.

Ex
```golang
func convertBep3CoinAmountToERC20Amount(amount *big.Int) (result *big.Int) {
	return new(big.Int).Mul(amount, bep3ConversionFactor)
}
func convertBep3ERC20AmountToCoinAmount(amount *big.Int) (quotient *big.Int) {
	return new(big.Int).Div(amount, bep3ConversionFactor)
}

Reply:
good point, i think this will benefit here